lib/sentry/plug.ex

```diff
@@ -171,24 +171,14 @@ if Code.ensure_loaded?(Plug) do
         headers: handle_data(conn, header_scrubber),
         env: %{
           "REMOTE_ADDR" => remote_address(conn.remote_ip),
-          "REMOTE_PORT" => remote_port(conn),
+          "REMOTE_PORT" => Plug.Conn.get_peer_data(conn).port,
```

Comment:
this function was added in https://github.com/getsentry/sentry-elixir/pull/326 as `Plug.Conn.get_peer_data` was returning an `:error` tuple, but I'm not sure if that's possible looking at Plug docs as it is now, and the typespecs seem to confirm that.

I'm going to assume it's good now though 🙂